src/utils/feeBreakdown.ts

```diff
@@ -47,41 +45,50 @@ const getCalculatedAmount = (data: PassportFeeFields) =>
 export const calculateReductionOrExemptionAmounts = (
   data: PassportFeeFields,
 ): ReductionOrExemption => {
-  const hasExemption =
-    data["application.fee.exemption.disability"] ||
-    data["application.fee.exemption.resubmission"];
+  const { reductions, exemptions } = getReductionOrExemptionLists(data);
+
+  const hasExemption = exemptions.length > 0;
   if (hasExemption) {
     return {
-      exemption:
-        data["application.fee.payable"] || data["application.fee.calculated"],
+      exemption: -getCalculatedAmount(data),
       reduction: 0,
     };
   }
 
+  const hasReductions = reductions.length > 0;
+  if (!hasReductions) {
+    return {
+      exemption: 0,
+      reduction: 0,
+    };
+  }
+
+  // Reductions should exclude and be calculated prior to extra VAT-able charges & fees being applied
+  const extraCharges =
+    (data["application.fee.serviceCharge"] || 0) +
+    (data["application.fee.fastTrack"] || 0) +
+    (data["application.fee.paymentProcessing"] || 0) +
+    (data["application.fee.payable.VAT"] || 0); // sum of all VAT
```

Comment:
The ` || 0`s should be redundant here as we're setting this as the default value.